src/assets/toolkit/scripts/lib/component/fact-slider.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+export class FactSlider {
+  constructor (element, {
+    facts = element.querySelectorAll('.FunFact'),
+    factNavNext = element.querySelector('.js-FunFact-next'),
```

Comment:
Nit: Your variable names are verbose in the wrong way. Consider describing what the function of the element is relative to this component... for example, `nextElement`, `nextButton`, `nextTrigger`, etc.